Unix/wsman/wsmanclient.c

```diff
@@ -573,14 +628,31 @@ void _WsmanClient_PostControl( _In_ Strand* self, _In_ Message* msg)
 }
 void _WsmanClient_Ack( _In_ Strand* self_)
 {
-//    WsmanClient* self = FromOffset( WsmanClient, strand, self_ );
+    WsmanClient* self = FromOffset( WsmanClient, strand, self_ );
 //    ProtocolBase* protocolBase = (ProtocolBase*)self->base.data;
     DEBUG_ASSERT( NULL != self_ );
 
     trace_ProtocolSocket_Ack( &self_->info.interaction, self_->info.interaction.other );
 //    if (!(self->base.mask & SELECTOR_WRITE))
 //       self->base.mask |= SELECTOR_READ;
 //    Selector_Wakeup(HttpClient_GetSelector(self->httpClient), MI_FALSE );
+
+    if (self->endOfSequence)
```

Comment:
Need a comment as to what is going on. EndOfSequence is only set to false for enumerate operations -- may want to make that comment in the declaration of this also